Makefile

```diff
@@ -72,6 +77,10 @@ build-migtator: test
 # Build all docker images for amd64
 build-amd64: build-op-amd64 build-migtator-amd64
 
+# Build the docker image for amd64
```

Comment:
Build the docker image for amd64
=>
Build amd64 Fluent Bit container image

Build the docker image for amd64
=>
Build amd64 Fluent Bit Operator container image